package.json

```diff
@@ -47,7 +47,7 @@
     "bunyan": "1.8.14",
     "codecov": "3.7.2",
     "deep-freeze": "0.0.1",
-    "graphql": "14.7.0",
+    "graphql": "15.5.0",
```

Comment:
Fwiw, I just updated Renovate to start actually bumping package dependencies on this repo.  **We should land this PR before this Saturday**, otherwise there will be a number of merge conflicts. 😉